### PR TITLE
Added Python Solution to Back & Forth (Bronze)

### PR DESCRIPTION
**Pull request by ryanchou-dev**
also commented out Benjamin Qi's code and added language section tag.

all test cases pass for this solution :)

### CODE REVIEW

**Comment by ryanchou-dev** [comment]
would the title be better as "Back and Forth" instead of "Back & Forth" since USACO lists it with an "and"?
image

**Review by dongliu0426 — APPROVED** [review]
cool

**Review by maggie-j-liu — APPROVED** [review]

File: solutions/usaco-857.mdx [hunk 1]
```diff
@@ -14,6 +14,46 @@ functions ...
...
+
+def transfer(day, a_tank, a_buck, b_tank, b_buck):
+	# Last day, add the amount of milk in the first tank.
+	if day == 4:
+		possible.add(a_tank)
+		return
+		
+	# This transfers every possible bucket from barn x to barn y.
+	for i in range(len(a_buck)):
+
```
Comment (SansPapyrus683) [inline solutions/usaco-857.mdx:36]:
```suggestion
```

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
actually p good for cp standards

**Review by caoash — APPROVED** [review]

**Pull request merged**